cmake/AddInputFileTests.cmake

```diff
@@ -5,6 +5,15 @@ spectre_define_test_timeout_factor_option(INPUT_FILE "input file")
 
 find_package(Python REQUIRED)
 
+# Environment variables for test
+set(_TEST_ENV_VARS "")
```

Comment:
What's the leading underscore about?

Reply:
This is how it's named in `SpectreSetupPythonPackage.cmake` 🤷‍♂️ Some people prefix local variables with leading underscores in CMake.